packages/talker_flutter/lib/src/controller/talker_view_controller.dart

```diff
@@ -7,9 +7,14 @@ import 'package:talker_flutter/talker_flutter.dart';
 class TalkerViewController extends ChangeNotifier {
   BaseTalkerFilter _filter = BaseTalkerFilter();
 
-  var _expandedLogs = true;
+  bool _expandedLogs = true;
   bool _isLogOrderReversed = true;
 
+  void init({bool expandedLogs = true, isLogOrderReversed = true}){
+    _expandedLogs = expandedLogs;
+    _isLogOrderReversed = isLogOrderReversed;
+  }
```

Comment:
Please make this variables initialization without init() method. 
Can we pass ``expandedLogs`` and ``isLogOrderReversed`` in TalkerViewController constructor ?